capstone/src/main/java/com/google/sps/search/servlets/NameTestDataServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.search.servlets;
+
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.Objects.User;
+import com.google.sps.search.CsvReader;
+
+@WebServlet("/name-test-data")
+public class NameTestDataServlet extends HttpServlet {
+
+  @Override
+  public void init() {
+    CsvReader reader = new CsvReader();
+    List<String> userInfo = reader.parseCorpus("../../data/user_data.csv");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    for (String str : userInfo) {
+      String[] split = str.split(" ");
+      String firstName = split[0];
+      String lastName = split[1];
+      String userId = firstName + "_" + lastName;
```

Comment:
This doesn't guarantee uniqueness if you have multiple users with the same first/last name

Reply:
+1 Since this is just test data, you don't need to do anything elaborate, maybe just a simple counter that increases with each name.

Also as an FYI in case you didn't know, while you're doing simple concatenation here and therefore not an issue, you'll typically want to combine strings using a StringBuilder instead of concatenating strings because it is inefficient:
https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java
https://dzone.com/articles/string-concatenation-performacne-improvement-in-ja
https://javapapers.com/java/java-string-vs-stringbuilder-vs-stringbuffer-concatenation-performance-micro-benchmark/#:~:text=Difference%20between%20String%2C%20StringBuilder%2C%20StringBuffer&text=Using%20'%2B'%20operator%20and%20String,concat%20are%20not%20the%20same.&text=%2B%20operator%20uses%20StringBuilder%20internally%20to,as%20below%20by%20Java%20compiler.